micro-rdk/src/common/conn/viam.rs

```diff
@@ -486,6 +480,14 @@ where
             self.provision().await;
         }
 
+        #[cfg(feature = "ota")]
+        {
+            if self.storage.has_ota_metadata() {
+                let metadata = self.storage.get_ota_metadata().unwrap_or_default();
+                log::info!("firmware version: {}", metadata.version);
+            }
+        }
```

Comment:
adding this here avoids complicating config monitor's ota stuff to get the right logging behavior.